apps/nextra/pages/en/build/smart-contracts/_meta.tsx

```diff
@@ -1,15 +1,123 @@
 export default {
-  "learn-move": {
-    title: "Learn Move",
+  "---get-started---": {
+    type: "separator",
+    title: "Get Started",
   },
-  "aptos-standards": {
+  "why-move": {
+    title: "Why Move?",
+  },
+  "create-package": {
+    title: "Create Package",
+  },
+  "compiling": {
+    title: "Compiling",
+  },
+  "testing": {
+    title: "Testing",
+  },
+  "deployment": {
+    title: "Deployment",
+  },
+  "---addressable-storage---": {
+    title: "Addressable Storage",
+    type: "separator"
+  },
+  "accounts": {
+    title: "Accounts",
+  },
+  "table": {
+    title: "Table",
+  },
+  "object": {
+    title: "Objects",
+  },
+  "resource-accounts": {
+    title: "Resource Accounts",
+  },
+  "---data-structures---": {
+    title: "Data Structures",
+    type: "separator"
+  },
+  "struct": {
+    title: "Struct",
+  },
+  "vector": {
+    title: "Vector",
+  },
+  "smart-vector": {
+    title: "Smart Vector",
+  },
+  "smart-table": {
+    title: "Smart Table",
+  },
+  "---aptos-standards---": {
+    type: "separator",
     title: "Aptos Standards",
   },
-  compiler_v2: {
-    title: "Move Compiler v2 (beta)",
+  "aptos-object": {
+    title: "Object",
+  },
+  "digital-asset": {
+    title: "Digital Asset (DA)",
+  },
+  "fungible-asset": {
+    title: "Fungible Asset (FA)",
+  },
+  "aptos-coin": {
+    title: "Coin (legacy)",
+  },
+  "aptos-token": {
+    title: "Token (legacy)",
+  },
+  "---examples---": {
+    type: "separator",
+    title: "Examples"
+  },
+  "---reference---": {
+    type: "separator",
+    title: "Reference",
+  },
+  "move-book": {
+    title: 'Move Book',
+  },
+  "move-reference": {
+    title: "Move Reference",
   },
-  // TODO: Add Move Reference - https://aptos.dev/reference/move
   "error-codes": {
-    title: "Move Error Codes",
+    title: "Error Codes"
+  },
+  "---advanced-move---": {
+    type: "separator",
+    title: "Advanced Move",
+  },
+  "scripts": {
+    title: "Scripts"
+  },
+  "patterns": {
+    title: "Patterns",
+  },
+  "move-prover": {
+    title: "Move Prover",
+  },
+  "cryptography": {
+    title: "Cryptography",
+  },
+  "move-security-guidelines": {
+    title: "Move Security Guidelines",
+  },
```

Comment:
Can we put at least redirect pages, or move existing pages for those that may already exist?

Reply:
I was gonna do that as a fast follow. I've rebased this a couple of times already because people are changing a bunch of the content in smart contracts and its led to a bunch of merge conflicts